### PR TITLE
Add module serving example using BentoML

### PR DESCRIPTION
**Pull request by cqvu**
Added an example ipynb of serving a PaddleHub module (deeplabv3p_xception65_humanseg) using [BentoML](github.com/bentoml/BentoML).

### CODE REVIEW

File: demo/serving/bentoml_serving/deeplabv3p_xception65_humanseg.ipynb [hunk 1]
```diff
@@ -0,0 +1,448 @@
+{
```
Comment (parano) [inline demo/serving/bentoml_serving/deeplabv3p_xception65_humanseg.ipynb]: thanks Chau, could we rename the folder from `bentoml_serving` to just `bentoml`? and rename notebook to `cloud-native-model-serving-with-bentoml.ipynb`?

Comment (cqvu) [reply demo/serving/bentoml_serving/deeplabv3p_xception65_humanseg.ipynb]: @parano renamed!

File: demo/serving/bentoml/cloud-native-model-serving-with-bentoml.ipynb [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+        "@env(infer_pip_packages=True)\n",
+        "class PaddleHubService(bentoml.BentoService):\n",
+        "    def __init__(self):\n",
+        "      super(PaddleHubService, self).__init__()\n",
+        "      self.module = hub.Module(name=\"deeplabv3p_xception65_humanseg\")\n",
+        "\n",
+        "    @api(input=ImageInput(), batch=True)\n",
+        "    def predict(self, image: imageio.core.Image):\n",
+        "        results = self.module.segmentation(images=[image], visualization=True)\n",
+        "        return results[0]['data']\n"
```
Comment (bojiang) [inline demo/serving/bentoml/cloud-native-model-serving-with-bentoml.ipynb]:
```suggestion
        "    @api(input=ImageInput(), batch=True)\n",
        "    def predict(self, images):\n",
        "        results = self.module.segmentation(images=images, visualization=True)\n",
        "        return [result['data'] for result in results]\n"
```

File: demo/serving/bentoml/cloud-native-model-serving-with-bentoml.ipynb [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "ondQXpNCy_TV"
+      },
+      "source": [
+        "# Predict with the initialized module\n",
+        "image = cv2.imread(\"test_image.jpg\")\n",
+        "segmentation_result = bento_svc.predict(np.array(image))"
```
Comment (bojiang) [inline demo/serving/bentoml/cloud-native-model-serving-with-bentoml.ipynb]:
```suggestion
        "segmentation_result = bento_svc.predict([image])"
```

File: demo/serving/bentoml/cloud-native-model-serving-with-bentoml.ipynb [hunk 4]
```diff
@@ -0,0 +1,448 @@
...
+      "cell_type": "code",
+      "metadata": {
+        "id": "jNnyhPQt59ey"
+      },
+      "source": [
+        "# View the segmentation mask layer\n",
+        "\n",
+        "# If running in a Google Colab notebook, run these lines\n",
+        "from google.colab.patches import cv2_imshow\n",
+        "cv2_imshow(segmentation_result)\n",
```
Comment (nepeplwu) [inline demo/serving/bentoml/cloud-native-model-serving-with-bentoml.ipynb]: `cv2_imshow` seems to be a specific interface of google colab, it is recommended to replace it with other common interfaces.

**Pull request merged**